js/bootstrap-timepicker.js

```diff
@@ -147,7 +147,20 @@
       if (step) {
         newVal = this.minute - step;
       } else {
-        newVal = this.minute - this.minuteStep;
+        var minuteStep = this.minuteStep;
+        if (this.minuteStep > 60) {
+          var hourStep = Math.floor(this.minuteStep / 60);
+          minuteStep -= (hourStep * 60);
+          if (minuteStep == 0) {
+            minuteStep = 60;
+          }
+
+          for (var i = 0; i<hourStep-(minuteStep == 60 ? 1 : 0); i++) {
```

Comment:
add whitespace around operators, like you see in the rest of the code (hopefully).

If you're worried about verbosity on this line, you can pull out `hourStep - (minuteStep === 60 ? 1 : 0)` into a variable declared in the line above.